Partition (WBP)/Functions/V56118C.VT156118.fun.yml

```diff
@@ -220,23 +265,33 @@
     #Com_Owner.mthAdd_SubMsg( OK ' + Matched results' '' )
     Endif
     Endif
-    Endroutine
     
-    *
-    *
-    *
-    Mthroutine Name(mthSelect_Where_IsNotSqlnull)
-    Define_Map For(*INPUT) Class(#F156118K1) Name(#this_key)
-    Define_Map For(*INPUT) Class(#Std_Count) Name(#this_expected_number_of_records)
-    Define_Map For(*BOTH) Class(#BIFA002A) Name(#this_rc)
+    * Boolean Column.
+    If (#this_rc <> ER)
+    #Com_Owner.mthAdd_SubMsg( OK '- ((#F156118K2 <> 0) *And (#BIFB001C.IsSqlnull))' '' )
+    #LF_i := 0
+    Select Fields(*ALL) From_File(VTL156118C) Where((#F156118K2 <> 0) *And (#F156118D.IsSqlnull)) With_Key(#this_key) Io_Status(#this_rc) Io_Error(*NEXT) Val_Error(*NEXT)
```

Comment:
You have not changed the WHERE clause in ALL of the SELECT commands. E.g. for this line the WHERE parameter should be:
```
(#F156118K2 <> 0) *And (#BIFB001C.IsSqlnull)
```
I note that the log message before the SELECT and the IF command to test the returned value are using the correct Field.
Please correct all the SELECT commands.